metricbeat/module/kubernetes/util/enrichers.md

```diff
@@ -0,0 +1,51 @@
+## Kubernetes Metadata enrichment
+
+[metadata diag](../_meta/images/enrichers.png)
+
+The metadata enrichment process involves associating contextual information, such as Kubernetes metadata (e.g., labels, annotations, resource names), with metrics and events collected by Elastic Agent and Beats in Kubernetes environments. This process enhances the understanding and analysis of collected data by providing additional context.
+
+### Key Components:
+
+1. **Metricsets:**
```

Comment:
To differentiate with datasets. Just writ something same term is called datasets in elastic-agent.
https://www.elastic.co/blog/an-introduction-to-the-elastic-data-stream-naming-scheme